src/v2/Components/Links/BackLink.tsx

```diff
@@ -1,27 +1,45 @@
 import React from "react"
-import { ChevronIcon, Flex, Text } from "@artsy/palette"
+import { BoxProps, ChevronIcon, Text, boxMixin } from "@artsy/palette"
 import { StyledLink } from "./StyledLink"
+import { RouterLinkProps } from "v2/Artsy/Router/RouterLink"
+import styled from "styled-components"
 
-interface BackLinkProps {
-  linkText: string
-  path: string
-}
+type BackLinkProps = RouterLinkProps & BoxProps
+
+const Container = styled(StyledLink)`
+  display: flex;
+  align-items: center;
+  ${boxMixin}
+`
 
-export const BackLink: React.FC<BackLinkProps> = ({ linkText, path }) => {
+export const BackLink: React.FC<BackLinkProps> = ({
+  children,
+  to,
+  ...rest
+}) => {
   return (
-    <>
-      <Flex flexDirection="row" alignItems="center" my={3}>
```

Comment:
For kind of 'generic' components like this: never include any margins or paddings that would effect how it sits in the parent context. Instead just extend or mixin `Box` and apply them from that external context.